lib/matplotlib/widgets.py

```diff
@@ -1930,61 +1930,60 @@ def toggle_selector(event):
-
-        and clear the drawn box/line. The ``pos_1`` and ``pos_2`` are
-        arrays of length 2 containing the x- and y-coordinate.
-
-        If *minspanx* is not *None* then events smaller than *minspanx*
-        in x direction are ignored (it's the same for y).
-
-        The rectangle is drawn with *rectprops*; default::
-
-          rectprops = dict(facecolor='red', edgecolor = 'black',
-                           alpha=0.2, fill=True)
-
-        The line is drawn with *lineprops*; default::
-
-          lineprops = dict(color='black', linestyle='-',
-                           linewidth = 2, alpha=0.5)
-
-        Use *drawtype* if you want the mouse to draw a line,
-        a box or nothing between click and actual position by setting
-
-        ``drawtype = 'line'``, ``drawtype='box'`` or ``drawtype = 'none'``.
-        Drawing a line would result in a line from vertex A to vertex C in
-        a rectangle ABCD.
-
-        *spancoords* is one of 'data' or 'pixels'.  If 'data', *minspanx*
-        and *minspanx* will be interpreted in the same coordinates as
-        the x and y axis. If 'pixels', they are in pixels.
-
-        *button* is the `.MouseButton` or list of `.MouseButton`\s used for
-        rectangle selection.  Default is *None*, which means any button.
-
-        *interactive* will draw a set of handles and allow you interact
-        with the widget after it is drawn.
-
-        *state_modifier_keys* are keyboard modifiers that affect the behavior
-        of the widget.
-
-        The defaults are:
-        dict(move=' ', clear='escape', square='shift', center='ctrl')
-
-        Keyboard modifiers, which:
-        'move': Move the existing shape.
-        'clear': Clear the current shape.
-        'square': Makes the shape square.
-        'center': Make the initial point the center of the shape.
-        'square' and 'center' can be combined.
+        """
+        Parameters
+        ----------
+        ax : `~matplotlib.axes.Axes`
+            The parent axes for the widget.
+        onselect : function
+            When a selection is made, the selector is cleared and *onselect*
+            is called with two arguments: the button press event and button
+            release event.
+        drawtype : {"box", "line", "none"}, default: "box"
+            Whether to draw the full rectangle box, the diagonal line of the
+            rectangle, or nothing at all.
+        minspanx : float, default: 0
+            Selections with an x-span less than *minspanx* are ignored.
+        minspany : float, default: 0
+            Selections with an y-span less than *minspany* are ignored.
+        useblit : bool, default: False
+            Whether to use blitting for faster drawing (if supported by the
+            backend).
+        lineprops : dict, default: \
+dict(color="black", linestyle="-", linewidth=2, alpha=0.5)
+            Properties with which the line is drawn, if ``drawtype == "line"``.
```

Comment:
```suggestion
        lineprops : dict, optional
            The properties of the line if ``drawtype='line'``. Default::

                lineprops = dict(linestyle='-', linewidth=2,
                                 color='black', alpha=0.5)
```
to not have to continue the definition line.